.travis.yml

```diff
@@ -17,18 +17,18 @@ deploy:
   - provider: s3
     access_key_id: $AWS_KEY
     secret_access_key: $AWS_SECRET
-    bucket: cru-givedev
```

Comment:
Is this bucket still around?

Reply:
Yes, this bucket still exists. There was a `cru-givetest` bucket as well that was empty, so I deleted it.